### ISSUE TITLE
vfide build fails with

### ISSUE THREAD

**Issue by necto** [verifast/verifast#111]
On my Ubuntu 16.04.3, on current `master` head:
```
$ make vfide
  OCAML     generate_vfversion.ml
File "branchleft_png.ml", line 36, characters 16-32:
Error: Unbound module GdkPixbuf
GNUmakefile:389: recipe for target 'branchleft_png.cmx' failed
make: *** [branchleft_png.cmx] Error 2
```
Here is the commit that breaks the build: 
https://github.com/verifast/verifast/commit/5f980c436be2f5f755f2c75276fe8b108d4c0e7f
What is the canonical way to build `vfide` on ubuntu now?

**Comment by btj** [comment]
Should be fixed by 6cad329a9518b0d721e3203b88369efde64f1be1 .

**Comment by necto** [comment]
Fixed. Thank you!
